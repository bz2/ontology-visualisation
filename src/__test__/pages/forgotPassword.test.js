import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Index from '../../pages/forgotPassword'

const setup = () => {
  const component = shallow(<Index />)

  return {
    component
  }
}

describe('Forgot Password page', () => {
  it('should match snapshot', () => {
    const { component } = setup()
    expect(toJson(component)).toMatchSnapshot()
  })
})
