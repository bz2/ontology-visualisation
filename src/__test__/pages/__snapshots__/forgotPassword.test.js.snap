// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forgot Password page should match snapshot 1`] = `
<main
  className="forgotPassword"
>
  <div
    className="forgotPassword-container"
  >
    <div
      className="logo"
    >
      <img
        alt="Highways England"
        src={
          Object {
            "process": [Function],
          }
        }
      />
    </div>
    <h1
      className="forgotPassword-title"
    >
      forgotPassword
    </h1>
    <div
      className="forgotPassword-input-container"
    >
      <label
        className="forgotPassword-label"
        htmlFor="email"
      >
        enterEmail
      </label>
      <div
        className="p-input-icon-left forgotPassword-input"
      >
        <i
          className="pi pi-user"
        />
        <InputText
          id="email"
          keyfilter={null}
          onChange={[Function]}
          onInput={null}
          onKeyPress={null}
          tooltip={null}
          tooltipOptions={null}
          type="email"
          validateOnly={false}
          value=""
        />
      </div>
    </div>
    <Button
      badge={null}
      badgeClassName={null}
      className="forgotPassword-button m-t-20"
      disabled={false}
      icon={null}
      iconPos="left"
      label="send"
      onClick={[Function]}
      tooltip={null}
      tooltipOptions={null}
    />
  </div>
</main>
`;
